home_work_18/style.css

```diff
@@ -0,0 +1,240 @@
+}
+
+.status-massage {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  margin: 25px 10px 25px 0px;
+  font-size: 13px;
+}
+
+.error {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 25px 10px 25px 0px;
+  font-size: 40px;
+  color: rgb(173, 54, 54);
+}
+
+#films {
+  min-height: 730px;
+  padding: 0 10px 20px 10px;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-column-gap: 10px;
+}
+
+.film-style {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 10px;
+  border-radius: 5px;
+  border: 1px solid rgb(43, 43, 43);
+  padding: 0 16px 0 8px;
+  font-size: 22px;
+  box-shadow: 0px 0px 10px -4px rgb(0, 0, 0);
+}
+
+.film-style:hover {
+  box-shadow: 0px 0px 10px -4px rgb(60, 131, 131);
+}
+
+.image {
+  height: 110px;
+  width: 90px;
+}
+
+.film-title {
+  padding-left: 20px;
+}
+.hiden {
+  visibility: hidden;
+}
+#film-details {
+  display: flex;
+  flex-direction: column;
+  padding: 20px 0px 10px 10px;
+  background: linear-gradient(rgb(64 64 64), rgb(0, 0, 0));
+}
+
+.film-item {
+  display: flex;
+  align-items: center;
+}
+
+.details-item {
+  display: flex;
+  flex-direction: row;
+  padding: 4px;
+}
+
+.btn-details {
+  margin-left: 15px;
+  border-radius: 5px;
+  border: none;
+  cursor: pointer;
+  background-color: rgb(0, 124, 133);
+  color: white;
+  justify-self: center;
+  transition: all 0.1s linear 0s;
+  height: 30px;
+}
+
+.btn-details:hover {
+  transform: scale(1.2);
+  box-shadow: 0px 0px 20px -2px rgb(60, 131, 131);
+}
+.order {
+  order: -1;
+}
+
+.movie-poster {
+  width: 200px;
+}
+
+.item-key {
+  color: rgb(142, 213, 235);
+  font-weight: 300;
+  text-transform: capitalize;
+  margin-right: 6px;
+}
+
+.pages-container {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+.page {
+  display: flex;
+  justify-content: center;
+  cursor: pointer;
+  transition: all 0.1s linear 0s;
+  padding: 4px 8px;
+  box-shadow: 0px 0px 10px -4px rgb(0, 0, 0);
+  border-radius: 5px;
+  min-width: 28px;
+}
+.page:hover {
+  box-shadow: 0px 0px 10px -4px rgb(60, 131, 131);
+}
+:target {
+  color: red;
+}
+@-webkit-keyframes swing {
+  15% {
+    -webkit-transform: translateY(5px);
+    transform: translateY(5px);
+  }
+  30% {
+    -webkit-transform: translateY(-5px);
+    transform: translateY(-5px);
+  }
+  50% {
+    -webkit-transform: translateY(3px);
+    transform: translateY(3px);
+  }
+  65% {
+    -webkit-transform: translateY(-3px);
+    transform: translateY(-3px);
+  }
+  80% {
+    -webkit-transform: translateY(2px);
+    transform: translateY(2px);
+  }
+  100% {
+    -webkit-transform: translateY(0);
+    transform: translateY(0);
+  }
+}
+#next {
+  color: aliceblue;
+  background: none;
+  border: none;
+  cursor: pointer;
+}
+#prev {
+  color: aliceblue;
+  background: none;
+  border: none;
+  cursor: pointer;
+}
+.currentPage {
+  background-color: rgb(0, 124, 133);
```

Comment:
It does not work

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/84010501/204880015-4fea21f6-1a03-4f15-b9b0-cacbe3b20d61.png">